.github/ISSUE_TEMPLATE/---relman-checklist.md

```diff
@@ -20,6 +20,7 @@ _Think something can be automated? Add it to [this doc](https://docs.google.com/
   - Merge to master
```

Comment:
wrt the wiki, do we still need the "Downloading builds from Taskcluster" section? I assume the emails will contain the links, right? Maybe it's better to leave them in there just in case?

Reply:
I was really conflicted on removing the "Downloading builds from Taskcluster" section :)
Since the wiki is version controlled, I'll remove the section since, yeah, the emails contain the links :+1: